MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -1635,7 +1641,7 @@ static MTLVertexFormat mvkAdjustFormatVectorToSize(MTLVertexFormat format, uint3
 
     // Multisampling - must ignore allowed bad pMultisampleState pointer if rasterization disabled
     if (_isRasterizing && pCreateInfo->pMultisampleState) {
-        plDesc.sampleCount = mvkSampleCountFromVkSampleCountFlagBits(pCreateInfo->pMultisampleState->rasterizationSamples);
+        plDesc.rasterSampleCount = mvkSampleCountFromVkSampleCountFlagBits(pCreateInfo->pMultisampleState->rasterizationSamples);
```

Comment:
This is not always available. It probably needs a magic property like the other ones in `MoltenVK/MoltenVK/OS` in case  this property isn't supported.

Reply:
I hear you. I looked into that when I made the change. Oddly, `rasterSampleCount` is documented both in header and docs as being supported from the beginning (iOS 8/macOS 10.11). I assume it was publicly available back then, but not documented for some unknown reason.

Since it's explicitly documented that way, I'm not expecting it to cause problems with either runtime on older OS's, or app store filters...but Apple is certainly not perfect in its consistency...so we'll see.